### PR TITLE
Add description + include fields for new resources

### PR DESCRIPTION
**Pull request by mlee-asana**
Followup from [this comment](https://github.com/Asana/api-explorer/pull/85#discussion_r871778661)

### CODE REVIEW

**Comment by jv-asana** [comment]
@mlee-asana do we want to fill in examples for:
```
        "type": "",
        "example_values": [],
        "comment": "",
```
?

I see these values filled out for [tasks](https://github.com/Asana/api-explorer/blob/master/src/resources/tasks.ts) for example. I guess we don't really use these values in our API Explorer UI?

File: src/resources/audit_log_api.ts [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+        "example_values": [],
+        "comment": "",
+    },
+    {
+        "name": "created_at",
+        "type": "",
+        "example_values": [],
+        "comment": "",
+    },
+];
```
Comment (jv-asana) [inline src/resources/audit_log_api.ts:28]: Aren't we missing a few properties here for [AuditLogEvent](https://developers.asana.com/docs/audit-log-event) like `details`, `event_category`, `event_type`, `resource`?

**Comment by mlee-asana** [comment]
> I see these values filled out for [tasks](https://github.com/Asana/api-explorer/blob/master/src/resources/tasks.ts) for example. I guess we don't really use these values in our API Explorer UI?

yep - the only field that is used is `name`, the other ones aren't used so didn't fill them out.

**Comment by jv-asana** [comment]
> > I see these values filled out for [tasks](https://github.com/Asana/api-explorer/blob/master/src/resources/tasks.ts) for example. I guess we don't really use these values in our API Explorer UI?
> 
> yep - the only field that is used is `name`, the other ones aren't used so didn't fill them out.

Cool. That makes sense.

File: src/resources/audit_log_api.ts (hunk 1 above)
Comment (mlee-asana) [reply src/resources/audit_log_api.ts:28]: I didn't do an exhaustive list for each resource, can probably go back and add that relatively easily

Comment (jv-asana) [reply src/resources/audit_log_api.ts:28]: Cool. If it's relatively easy to add let's go ahead and add them.

Comment (mlee-asana) [reply src/resources/audit_log_api.ts:52]: done!

**Review by jv-asana — APPROVED** [review]
LGTM. Thank you for taking time to add the exhaustive list of properties 🙂

**Pull request merged**